package.json

```diff
@@ -91,16 +101,17 @@
     "reactstrap": "~8.0.0",
     "redux": "~3.7.2",
     "redux-thunk": "~2.2.0",
-    "request": "~2.88.0",
+    "request": "~2.88.2",
     "request-promise-native": "~1.0.7",
     "serve-favicon": "~2.5.0",
-    "uglifyjs-webpack-plugin": "~1.2.4",
-    "webpack": "~4.30.0",
-    "webpack-cli": "~3.1.2",
+    "terser-webpack-plugin": "~5.0.0",
+    "watchpack": "~2.1.1",
+    "webpack": "~5.45.1",
+    "webpack-cli": "~4.9.0",
+    "webpack-dev-server": "~4.5.0",
```

Comment:
This comment is for an unchanged line below for @types/plotly.js. 1.41.0 is 3 years old. As long as we are updating plotly maybe we should update this too?

Reply:
moved to 1.54.12